### PR TITLE
Fix #15864: Table comments being required

### LINKED ISSUE #15864: Table comment shouldn't be required
pma

V: 5.1.0-dev

### PR DESCRIPTION
**Pull request by rabeehrz**
Signed-off-by: Mohammed Rabeeh <mohammadrabeeh@gmail.com>

### Description

Fixes #15864

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
I will do some research to find the commit that added this issue :)

**Comment by williamdes** [comment]
30b5fc46d2f68b826ab838faecab72ab0dab5aa4

**Comment by rabeehrz** [comment]
> I will do some research to find the commit that added this issue :)

@williamdes I think the initial commit was the one. File history doesn't show any other commit. 
[Commit](https://github.com/phpmyadmin/phpmyadmin/commit/30b5fc46d2f68b826ab838faecab72ab0dab5aa4#diff-243af6a540f547f30180d431b48e4233)

**Comment by williamdes** [comment]
You had forgot a quote, I added it in b47ea134cc9268d170e9b9048cb945611621ae88

Thank you for the quick fix :)

File: templates/table/operations/index.twig [hunk 1]
```diff
@@ -102,7 +102,7 @@
 
       <div class="form-group form-inline">
         <label for="tableCommentsInput">{% trans 'Table comments' %}</label>
-        <input class="form-control" id="tableCommentsInput" type="text" name="comment" maxlength="2048" value="{{ table_comment }}" required>
+        <input class="form-control" id="tableCommentsInput" type="text" name="comment" maxlength="2048" value="{{ table_comment }}>
```
Comment (williamdes) [inline templates/table/operations/index.twig]:
```suggestion
        <input class="form-control" id="tableCommentsInput" type="text" name="comment" maxlength="2048" value="{{ table_comment }}">
```

b47ea134cc9268d170e9b9048cb945611621ae88

**Pull request merged**